src/BaseStrategy.js

```
@@ -0,0 +1,80 @@
+const escape = require('lodash/escape');
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)